orttraining/orttraining/core/graph/gradient_schema_defs.cc

```diff
@@ -473,6 +478,11 @@ void RegisterGradientSchemas() {
           "Small scalar to avoid dividing by zero.",
           AttributeProto::FLOAT,
           1e-8f)
+      .Attr(
+          "do_bias_correction",
+          "Compute unbiased 1st and 2nd momentums.",
+          AttributeProto::INT,
+          static_cast<int64_t>(1))
```

Comment:
I am wondering if we need to make this a enum?
Just like apex does... 

typedef enum{
  ADAM_MODE_0   =0, // L2 regularization mode
  ADAM_MODE_1   =1  // Decoupled weight decay mode(AdamW)
} adamMode_t;

Reply:
Is TJ's weight decay variation going to be another flag ?

---
In reply to: [397589273](https://github.com/microsoft/onnxruntime/pull/3301#discussion_r397589273) [](ancestors = 397589273)

Reply:
Nevermind, this flag should stand alone ... 

---
In reply to: [397590509](https://github.com/microsoft/onnxruntime/pull/3301#discussion_r397590509) [](ancestors = 397590509,397589273)

Reply:
Yes. Weight decay is another flag. They are independent and compatible with each other.